FileManager/FileManager/ViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewController.swift
+//  FileManager
+//
+//  Created by Abdullah Alnutayfi on 23/11/2021.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    var folders : [String] = []
+    var selectedFolder = ""
+  
+    let textField = UITextField()
+    let tableView = UITableView()
+    let textView = UITextField()
+    let button1 = UIButton()
+    let button2 = UIButton()
```

Comment:
do you know that you can create your element with all of its attributes in one place? 
you can use a closer for that! 
let me show you, 
```swift
    lazy var yourButton : UIButton = {
        $0.translatesAutoresizingMaskIntoConstraints = false
        $0.backgroundColor = UIColor.blue
        $0.setTitle("Create Folder", for: .normal)
        $0.setTitleColor(.white, for: .normal)
        $0.addTarget(self, action: #selector(createFolderBtnClick), for: .touchDown)
        
        return $0
    }(UIButton(type: .system))
```

now you can use `yourButton` everywhere! but don't forget to add it to your view + giving it some constraints